nilearn/image/image.py

```diff
@@ -1631,9 +1669,11 @@ def concat_imgs(
 
     Parameters
     ----------
-    niimgs : iterable of Niimg-like objects or glob pattern
+    niimgs : iterable of Niimg-like objects, or glob pattern, \
```

Comment:
I guess the parameter could also be updated to `imgs`?

Reply:
yeah but that means a deprecation cycle

before I go full on to update the doc strings and other things, I wanted to make sure we are all ok with this move in principle